tools/deb/Makefile

```diff
@@ -0,0 +1,2 @@
+
+deb:
```

Comment:
empty?

Reply:
Fixed (removed, I put the deb in the top makefile )